__tests__/integration.js

```diff
@@ -60,7 +60,8 @@ addTest('https://github.com/yarnpkg/yarn/releases/download/v0.18.1/yarn-v0.18.1.
 addTest('https://github.com/bestander/chrome-app-livereload.git'); // no package.json
 addTest('bestander/chrome-app-livereload'); // no package.json, github, tarball
 // Only run `react-scripts` test on Node 6+
-if (parseInt(process.versions.node.split('.')[0], 10) >= 6) {
+const nodeMajorVersion = parseInt(process.versions.node.split('.')[0], 10);
```

Comment:
We still support Node 4 AFAIK. I think a better option would be to add the `--no-node-version-check` option to yarn invocations.